CONTRIBUTING.rst

```diff
@@ -8,6 +8,7 @@ If you like the project and think you could help with making it better, there ar
 - Triage old issues that needs a refresh
 - Implement existing issues (there are quite some of them, choose whatever you like)
 - Help with improving the documentation (We still have work to do!)
+- Help with improving the CLI documentation at :py:func:`pydoctor.driver.getparser`
```

Comment:
btw... this can be removed, as I have introduced another intershphinx link in usage.rst

Reply:
```suggestion
```